client/my-sites/customer-home/cards/tasks/deprecate-editor/index.jsx

```diff
@@ -0,0 +1,101 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import { preventWidows } from 'lib/formatting';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import Task from '../task';
+
+import { setSelectedEditor } from 'state/selected-editor/actions';
+import {
+	composeAnalytics,
+	recordGoogleEvent,
+	recordTracksEvent,
+	withAnalytics,
+	bumpStat,
+} from 'state/analytics/actions';
+import getGutenbergEditorUrl from 'state/selectors/get-gutenberg-editor-url';
+import blockEditorImage from 'assets/images/illustrations/block-editor-fade.svg';
+import FormattedDate from 'components/formatted-date';
+import { localizeUrl } from 'lib/i18n-utils';
+import InlineSupportLink from 'components/inline-support-link';
+
+const DeprecateEditor = ( { siteId, gutenbergUrl, optIn } ) => {
+	const translate = useTranslate();
+	const actionCallback = () => {
+		optIn( siteId, gutenbergUrl );
+	};
+	return (
+		<Task
+			title={ translate( 'The Block Editor is coming.' ) }
+			description={ preventWidows(
+				translate(
+					'Get a head start before we enable it for everyone on {{strong}}{{date}}{{/date}}{{/strong}}. {{support}}{{/support}}',
```

Comment:
This string can be tidied up a bit to make it easier to translate:

https://github.com/Automattic/wp-calypso/pull/41391/files#diff-b1e45afdbbd98883cd588ac9e267cd74R82

Reply:
ah, yeh, that is a better approach, thanks, done.